src/utils/maskHandler.js

```diff
@@ -26,15 +26,15 @@ export default function (fields) {
 
       case 'date':
         element.setAttribute('placeholder', '__/__/____');
-        const dateMask = IMask(element, maskDate());
+        var dateMask = IMask(element, maskDate());
         element.addEventListener('input', function () {
           dateMask.updateValue();
         });
         break;
 
       case 'time':
         element.setAttribute('placeholder', '__:__');
-        const timeMask = IMask(element, maskTime());
+        var timeMask = IMask(element, maskTime());
```

Comment:
Por que a mudança de const pra var?

Reply:
Depois que eu coloquei o eslint com a standard eslint config ele deu esse erro: `Unexpected lexical declaration in case block.eslintno-case-declarations`